### ISSUE TITLE
[ISSUE for v2]:

### ISSUE THREAD

**Issue by csam10398** [w-okada/voice-changer#1474]
### Voice Changer Version

vcclient_mac_2.0.76-beta.zip

### Operational System

MacOS 15

### GPU

None

### CUDA Version

None

### Read carefully and check the options

- [ ] If you use win_cuda_torch_cuda edition, setup cuda? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [ ] If you use win_cuda edition, setup cuda and cudnn? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [ ] If you use mac edition, client is not launched automatically. Use chrome to open application.?
- [x] I've tried to change the Chunk Size
- [x] I've tried to set the Index to zero
- [x] I've read the [tutorial](https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md)
- [x] I've tried to extract to another folder (or re-extract) the .zip file

### Does pre-installed model work?

No

### Model Type

RVC

### Issue Description

When I am speaking, its pronouncation heard wrongly. like growling or hissing or sound of vomitting.

### Application Screenshot

<img width="738" alt="Image" src="https://github.com/user-attachments/assets/6ad8b250-6534-4c6f-90b4-3f3304fa1c31" />

### Logs on console

/Users/josanghui/Downloads/dist/start_http.command ; exit;                      
(base) josanghui@josanghuiui-MacBookPro ~ % /Users/josanghui/Downloads/dist/start_http.command ; exit;
launching vcclient...
working directory: /Users/josanghui/Downloads/dist
checking the modules...

    vcclient    

----------------------------------------------------------
 Application | http://localhost:18000 
 Log(rich)   | http://localhost:18000/?app_mode=LogViewer 
 Log(text)   | http://localhost:18000/vcclient.log 
 API         | http://localhost:18000/docs 
 License(js) | http://localhost:18000/licenses-js.json 
 License(py) | http://localhost:18000/licenses-py.json 
----------------------------------------------------------
Please press Ctrl+C once to exit vcclient.
torch/nn/utils/weight_norm.py:28: UserWarning: torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.
  warnings.warn("torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.")
torch/nn/functional.py:4522: UserWarning: MPS: The constant padding of more than 3 dimensions is not currently supported natively. It uses View Ops default implementation to run. This may have performance implications. (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/native/mps/operations/Pad.mm:472.)

**Comment by csam10398** [comment]
PLEASE HELP ME HURRILY!!!!!(?)

**Comment by Kuuko-fokkusugaru** [comment]
lol does it happens the same with the included models?
Could you record a little short clip of video where I could listen how exactly sounds? In that way I may be able to identify what could be the issue

**Comment by csam10398** [comment]
I firgured the detail:
my earbuds doesn't have any error(Voice Recorder App)

Yet, 'in' and 'out' recorded like ROARing sound. I can't record the voice using quicktime player. How to record the speaker sound and the screen at the same time?

**Comment by Kuuko-fokkusugaru** [comment]
I am not a Mac user so I am not sure. But I have noticed that you set the same output for monitor and output, you can't do this as it's redundant. Set monitor to none until you are going to set output to something else. Also try setting the pitch to something else. For female to male conversion, use negative numbers. For male to female use positive numbers. For male to male or female to female, it can vary depending on how deep or high your voice is vs the model. If the model is deeper than your voice set it to negative numbers, if it's higher then set to positive numbers.
